package/MDAnalysis/coordinates/PDB.py

```diff
@@ -907,15 +907,15 @@ def write_all_timesteps(self, obj):
 
         for framenumber in range(start, len(traj), step):
             traj[framenumber]
-            self.write_next_timestep(self.ts, multiframe=True)
+            self._write_next_frame(self.ts, multiframe=True)
 
         self._write_pdb_bonds()
         self.close()
 
         # Set the trajectory to the starting position
         traj[start]
 
-    def write_next_timestep(self, ts=None, **kwargs):
+    def _write_next_frame(self, ts=None, **kwargs):
```

Comment:
Is this the only `_write_next_frame` that takes a Timestep?

Reply:
I think so yes, it's because PDB also defines its own `write()` method